src/homework9/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,106 @@
+package homework9.linkedlist;
+
+public class LinkedList<T> implements List<T> {
+    private Node<T> node;
+    private int size;
+    private Node<T> prev;
+    private Node<T> first;
+    private Node<T> last;
+
+    public LinkedList() {
+        this.size = 0;
+    }
+
+    @Override
+    public void add(T value) {
+        if (size == 0) {
+            node = first = last = new Node(null, value, null);
```

Comment:
do we need node here?

Reply:
I think no.

Reply:
... but I need this node to simplify method